client/src/components/common/Form.jsx

```diff
@@ -1,8 +1,59 @@
 import styled from '@emotion/styled';
+import { Flex, FormLabel, Icon, Input, Box } from '@chakra-ui/react';
+import { Text } from '../common/Typography.jsx';
 
 export const StyledForm = styled.form`
   width: 100%;
   display: flex;
   flex-direction: column;
   align-items: center;
 `;
+
+export const ValidationText = styled(Text)`
+  font-size: 10px;
+  padding-left: 12px;
+  width: 400px;
+`;
+
+export const Container = styled(Box)`
+  width: 100%;
+  height: 456px;
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  left: 50%;
+  align-items: center;
+  transform: translate(-50%, -50%);
+  background-color: white;
+  padding: 10px;
+  max-width: 560px;
+`;
```

Comment:
When you merged, you left out stuff that Karin and Sujan just wrote.

Use this as reference: https://github.com/Mackbellemore/soen-390-team07/commit/60ddee222055e7bc028ef93f5dc57f503d3f11bf

Reply:
I moved every new CSS inside the Form page with the change to existing CSS

Reply:
also the new component inside Login page, forgot to mention that